lib/webdriver-plus.ts

```diff
@@ -113,26 +121,48 @@ class WebElementRect implements ClientRect {
   get right(): number { return this.rect.x + this.rect.width; }
 }
 
-async function findContentHelper(driver: WebDriver, finder: WebElement|null,
-                                 selector: string, contentRE: RegExp): Promise<WebElement> {
+async function findContentHelper(driver: WebDriver, finder: WebElement|null, selector: string,
+                                 contentRE: RegExp, throwErrOnMiss: boolean = true): Promise<WebElement> {
```

Comment:
It would be a bit cleaner to have two helpers: `findContentIfPresent(...)` which calls executeScript and returns an element or null; and `findContentHelper()` which behaves as before, but simply calls `findContentIfPresent` and throws a `new NoSuchElementError("No elements match ${selector} and ${contentRE}")`. In fact, it seems a better way to handle it than catch and rethrow an error!